backend/filters.py

```diff
@@ -64,20 +62,20 @@ def apply_filter(self) -> pd.DataFrame:
 class FilterManager:
     _df: pd.DataFrame
 
-    def __init__(self, df_to_filter: pd.DataFrame):
+    def __init__(self, df_to_filter: pd.DataFrame, filters: list[Filter]):
         self._df = df_to_filter.copy()
+        self.filters = filters
```

Comment:
How does this run? filters should be added as an instance variable under the class FilterManager/

Reply:
RIght, filters will be an instance variable then in FilterManager.apply_filters() all of the subclasses of Filter stored in self.filters will get checked and one will get applied